examples/chat/hub.go

```diff
@@ -40,9 +46,13 @@ func (h *Hub) run() {
 				close(client.send)
 			}
 		case message := <-h.broadcast:
+			data, err := json.Marshal(message)
+			if err != nil {
+				log.Print(fmt.Sprint("Error while marshalling message:", err))
```

Comment:
The application blunders ahead after the error.  Error handing is more than logging a message.